doc/user-guide/06-layout-configuration.adoc

```diff
@@ -143,61 +106,60 @@ we mount the +/dev/mapper/backup-backup+ filesystem on
 +/media/backup+, it will get excluded, as the mountpoint is under
 +/media+.
 
-=== Manual excludes
-It seems prudent to prevent the external drives from ever being backed-up or overwritten. The default configuration contains these lines:
+If we mount the filesystem on +/dev/mapper/backup-backup+ on +/backup+,
+Relax-and-Recover will think that it's necessary to recreate the filesystem:
 ----------------------------------
-# Exclude components from being backed up, recreation information is active
-EXCLUDE_BACKUP=()
-
-# Exclude components during component recreation
-# will be added to EXCLUDE_BACKUP (it is not backed up)
-# recreation information gathered, but commented out
-EXCLUDE_RECREATE=()
-
-# Exclude components during the backup restore phase
-# only used to exclude files from the restore.
-EXCLUDE_RESTORE=()
+disk /dev/sdc 1999696297984 msdos
+...
+part /dev/sdc 1073741824000 1048576 primary boot /dev/sdc1
+part /dev/sdc 925953425408 1073742872576 primary lvm /dev/sdc2
+lvmdev /dev/backup /dev/sdc2 cJp4Mt-Vkgv-hVlr-wTMb-0qeA-FX7j-3C60p5 1808502784
+...
+lvmgrp /dev/backup 4096 220764 904249344
+...
+lvmvol /dev/backup backup 12800 104857600 
+lvmvol /dev/backup externaltemp 38400 314572800 
+...
+fs /dev/mapper/backup-backup /backup ext4 uuid=da20354a-dc4c-4bef-817c-1c92894bb002 label= blocksize=4096 reserved_blocks=655360 max_mounts=24 check_interval=180d options=rw
 ----------------------------------
 
-To prevent an inadvertently mounted backup filesystem being added to the restore list, the easiest way is to add the filesystem to the +EXCLUDE_RECREATE+ array.
+=== Manual excludes
+It seems prudent to prevent the external drives from ever being backed-up or overwritten.
+The most generic mechanism for doing this is the +EXCLUDE_RECREATE+
```

Comment:
> why a superlative judgement here?

because there are less generic inclusion/exclusion rules in ReaR, like `EXCLUDE_VG` or `ONLY_INCLUDE_VG`, and `EXCLUDE_RECREATE` is the most generic of them (can exclude anything).

Reply:
Is there something that is even more specific as EXCLUDE_VG?
So we would have
* EXCLUDE_RECREATE is most generic
* EXCLUDE_VG is generic
* ??? is specific

This is perhaps a stupid question
but I really have understanding problems
with all that include/exclude stuff in ReaR.

Reply:
@jsmeix I don't think so. There is `EXCLUDE_MOUNTPOINTS`, which one could consider either more or less specific as `EXCLUDE_VG` as its scope is different.
The point of the superlative is that there is nothing more generic than `EXCLUDE_RECREATE` (which can include any component), but perhaps even that is not literally correct, as `AUTOEXCLUDE_PATH` could be considered more genrric in some sense (can exclude only filesystems, but their mountpoints can be specified as a directory prefix).

Reply:
@jsmeix addressed in bfad3f3207fefa8766fe0866ed21ca50a9b604b8

Reply:
See also https://github.com/rear/rear/issues/3129